catalog/transformer/catalog-transformer-rtf/src/test/java/ddf/catalog/transformer/output/rtf/BaseTestConfiguration.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.transformer.output.rtf;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import ddf.catalog.data.Attribute;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.types.Core;
+import ddf.catalog.transformer.output.rtf.model.ExportCategory;
+import ddf.catalog.transformer.output.rtf.model.RtfCategory;
+import java.util.Arrays;
+import java.util.Base64;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public abstract class BaseTestConfiguration {
+
+  static final String EMPTY_ATTRIBUTE = "Empty";
+  static final String SIMPLE_ATTRIBUTE = "Simple";
+
+  private static final String UNKNOWN_ATTRIBUTE = "Unknown";
+  private static final String BASE64_IMAGE =
```

Comment:
Since it's only one image that we would likely never update creating another resource would be more trouble than it's worth plus this way there is a minor advantage in not having file i/o.